### PR TITLE
Editorial: Update Maintainers Link.

### PR DESCRIPTION
**Pull request by snuggs**
Fixes https://github.com/whatwg/meta/issues/64

  Addresses:
    - https://github.com/whatwg/meta/pull/69
    - https://github.com/whatwg/meta/issues/65

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,4 +85,4 @@ double quotes).
 ### Merge policy
 
 If you can commit to this repository, see the
-[WHATWG team](https://github.com/whatwg/meta/blob/master/TEAM.md) guidelines.
+[WHATWG team](https://github.com/whatwg/meta/blob/master/MAINTAINERS.md) guidelines.
```
Comment (snuggs) [inline README.md]: Should we make this `WHATWG maintainers` ?

Comment (annevk) [reply README.md]: Sounds good to me. @domenic?

Comment (domenic) [reply README.md]: Sounds good

Comment (snuggs) [reply README.md]: Fixed in https://github.com/whatwg/infra/pull/186/commits/93cc2ed895522f0dad1731b9ad311ebca7ba6bb4

**Pull request merged**